ghost/core/core/server/web/comments/routes.js

```diff
@@ -22,14 +21,17 @@ module.exports = function apiRoutes() {
 
     router.get('/', http(api.commentsMembers.browse));
     router.get('/:id', http(api.commentsMembers.read));
```

Comment:
I think these use member authentication to determine if a comment has been liked?

Reply:
Ugh - I think you're right, I didn't see it because it's transparently passed all the way into the model layer 😢

Reply:
I've updated this to only exclude comment counts